expression/expression.go

```diff
@@ -77,6 +84,54 @@ func (triggerExpression TriggerExpression) Get(name string) (interface{}, error)
 	}
 }
 
+// Validate returns error if triggers of type moira.ExpressionTrigger are badly formatted otherwise nil
+func (triggerExpression *TriggerExpression) Validate() error {
+	if triggerExpression.TriggerType != moira.ExpressionTrigger {
```

Comment:
I think it will be better to use guard checks as there are many things we have already started guarding against inside the function. Like so:
```
	if triggerExpression.TriggerType != moira.ExpressionTrigger {
		return nil
	}
	if triggerExpression.Expression == nil || *triggerExpression.Expression == "" {
		return fmt.Errorf("trigger_type set to expression, but no expression provided")
	}
```

Reply:
ok, no problem